README.txt

```diff
@@ -1,21 +1,24 @@
-Methodocracy
+Methodocracy (noun):
+A cultural-political ideology which proposes that optimal decision-making, from the individual to national level, is necessarily based upon the quantity and quality of data, interpretation, and conclusions gathered through tested and reliable methodologies.
```

Comment:
Methodocracy is not a word coined by our organization, it is a brand name. The populous can interpret it and use it in any way they want. It only insinuates,